spring-cloud-dataflow-server-core/pom.xml

```diff
@@ -50,6 +50,10 @@
 			<groupId>org.hibernate.orm</groupId>
 			<artifactId>hibernate-micrometer</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate.orm</groupId>
+			<artifactId>hibernate-ant</artifactId>
```

Comment:
Uhhh what is this?

Reply:
This is used by SchemaGenerationTests for the `org.hibernate.tool.hbm2ddl.SchemaExport`.   I changed the scope of this to `test` and moved it under the tests in the dependencies.